### ISSUE TITLE
how do I make the new windows bg transparent without making the dad transparent?

### ISSUE THREAD

**Issue by TheRealJake12** [DuskieWhy/Transparent-and-MultiWindow-FNF#1]
doing the transparent function in the popup window function doesnt do anything and when you do the transparent function when the popup function is called it freezes the game

**Comment by TheRealJake12** [comment]
also there seems to be a VRAM memory leak. It just goes up when the popup window is used

**Comment by MemeHoovy** [comment]
> also there seems to be a VRAM memory leak. It just goes up when the popup window is used

I don't think this is finished, perhaps

**Comment by TheRealJake12** [comment]
yes perhaps. But it is a major VRAM memory leak. It just goes up until the game crashes.

**Comment by MemeHoovy** [comment]
> yes perhaps. But it is a major VRAM memory leak. It just goes up until the game crashes.

I haven't tried this before, so idk what could be causing that. Maybe you know?

**Comment by TheRealJake12** [comment]
¯ \ _ (ツ) _ / ¯

**Comment by MemeHoovy** [comment]
> ¯_(ツ)_/¯

I'll take that as a no

**Comment by 609135** [comment]
If you want to know how to make the character window transparent, just make an image that has the same resolution as your window and with the same rgb color as in the transparency script in (assets/shared/images) and name it whatever you want. Then set it as the bg of the window by following the steps in the video (https://www.youtube.com/watch?v=NZAmVfqN4cQ). After that, go ahead and paste the window transparency function that you made for the game's window (FlxTransWindow.getWindowsTransparent();) and that should work.

**Comment by MemeHoovy** [comment]
> If you want to know how to make the character window transparent, just make an image that has the same resolution as your window and with the same rgb color as in the transparency script in (assets/shared/images) and name it whatever you want. Then set it as the bg of the window by following the steps in the video (https://www.youtube.com/watch?v=NZAmVfqN4cQ). After that, go ahead and paste the window transparency function that you made for the game's window (FlxTransWindow.getWindowsTransparent();) and that should work.

<img width="340" alt="621 (10)" src="https://user-images.githubusercontent.com/97995998/189558751-52e7169b-6ce2-451c-8b3d-31f3a7693437.png">

fr though, it's pretty helpful though

**Comment by 609135** [comment]
> > If you want to know how to make the character window transparent, just make an image that has the same resolution as your window and with the same rgb color as in the transparency script in (assets/shared/images) and name it whatever you want. Then set it as the bg of the window by following the steps in the video (https://www.youtube.com/watch?v=NZAmVfqN4cQ). After that, go ahead and paste the window transparency function that you made for the game's window (FlxTransWindow.getWindowsTransparent();) and that should work.
> 
> <img alt="621 (10)" width="340" src="https://user-images.githubusercontent.com/97995998/189558751-52e7169b-6ce2-451c-8b3d-31f3a7693437.png">
![63syp7](https://user-images.githubusercontent.com/79272558/189561305-58f271a8-f504-435e-8a48-f4b719a408a7.jpg)

LISTEN I WROTE THAT IN LIKE 2 SECONDS ANDS FLKSDFJLSDK FJSDLK FJS

**Comment by TheRealJake12** [comment]
> If you want to know how to make the character window transparent, just make an image that has the same resolution as your window and with the same rgb color as in the transparency script in (assets/shared/images) and name it whatever you want. Then set it as the bg of the window by following the steps in the video (https://www.youtube.com/watch?v=NZAmVfqN4cQ). After that, go ahead and paste the window transparency function that you made for the game's window (FlxTransWindow.getWindowsTransparent();) and that should work.

give a screenshot of your code because I tried already

**Comment by 609135** [comment]
> > If you want to know how to make the character window transparent, just make an image that has the same resolution as your window and with the same rgb color as in the transparency script in (assets/shared/images) and name it whatever you want. Then set it as the bg of the window by following the steps in the video (https://www.youtube.com/watch?v=NZAmVfqN4cQ). After that, go ahead and paste the window transparency function that you made for the game's window (FlxTransWindow.getWindowsTransparent();) and that should work.
> 
> give a screenshot of your code because I tried already

![image](https://user-images.githubusercontent.com/79272558/189687642-ceea563a-bc16-4c5c-8a2f-59a9a50aeaae.png)

**Comment by TheRealJake12** [comment]
![image](https://user-images.githubusercontent.com/84357907/190014231-58008a97-887d-4a85-9938-fb92d8a5d452.png)
![image](https://user-images.githubusercontent.com/84357907/190014340-7f0d0630-6f49-46aa-8b7d-a6a5a03d626a.png)
nope

**Comment by TheRealJake12** [comment]
maybe its because I have the adds in a wrong place but that didnt work either
